include/mata/nfa.hh

```diff
@@ -39,6 +39,17 @@
 #include <mata/inter-aut.hh>
 #include <mata/synchronized-iterator.hh>
 
+/**
+ * Nondeterministic Finite Automata including structures, transitions and algorithms.
+ *
+ * In particular, this includes:
```

Comment:
The same as in AFA case. We should restructure file according this points and introduce the sections directly in comments in .code